travis/script-gradle

```diff
@@ -8,7 +8,7 @@ case "$(javac -version 2>&1)" in
   (javac\ 1.8.*)
      # linters only need to run on one operating system
      if [ "${TRAVIS_OS_NAME:-}" = linux ]; then
-       run_gradle -PjavaCompiler=ecj linters compileJava compileTestJava javadoc
+       run_gradle -PjavaCompiler=ecj linters compileJava compileTestJava compileTestFixturesJava compileTestSubjectsJava javadoc
```

Comment:
Could you update `CONTRIBUTING.md` to reflect the change in this command?  To avoid the need for this consistency check, it would be nice to create a top-level Gradle linting task that enables ECJ and runs all of these listed tasks.  But that can happen in a follow-up PR.

Reply:
Thanks for catching that. Added in a552e86ceb92d037d48792e9ea594e54fc8de7ec.